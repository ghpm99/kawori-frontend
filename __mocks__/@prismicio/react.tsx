const react = {
    SliceZone: (props) => (<div>SliceZone</div>)
};

module.exports = react;