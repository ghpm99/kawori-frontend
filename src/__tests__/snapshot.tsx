import { render } from "@testing-library/react";
import Cards from "@/components/overview/cards";

it("renders overview cards", () => {
    const { container } = render(
        <Cards
            loading={false}
            amountPayment={1508.15}
            amountPaymentClosed={2718.15}
            amountPaymentOpen={1508.15}
            countPayment={1508.15}
        />,
    );
    expect(container).toMatchSnapshot();
});
