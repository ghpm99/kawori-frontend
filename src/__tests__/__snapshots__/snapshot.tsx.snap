// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders overview cards 1`] = `
<div>
  <div
    class="cards-container"
  >
    <div
      class="ant-card ant-card-bordered css-dev-only-do-not-override-xu9wm8"
      style="flex-grow: 1;"
    >
      <div
        class="ant-card-head"
      >
        <div
          class="ant-card-head-wrapper"
        >
          <div
            class="ant-card-head-title"
          >
            Total de pagamentos
          </div>
        </div>
      </div>
      <div
        class="ant-card-body"
      >
        <div
          data-testid="countPayment"
        >
          1508.15
        </div>
      </div>
    </div>
    <div
      class="ant-card ant-card-bordered css-dev-only-do-not-override-xu9wm8"
      style="flex-grow: 1;"
    >
      <div
        class="ant-card-head"
      >
        <div
          class="ant-card-head-wrapper"
        >
          <div
            class="ant-card-head-title"
          >
            Valor total de pagamentos
          </div>
        </div>
      </div>
      <div
        class="ant-card-body"
      >
        <div
          data-testid="amountPayment"
        >
          R$ 1.508,15
        </div>
      </div>
    </div>
    <div
      class="ant-card ant-card-bordered css-dev-only-do-not-override-xu9wm8"
      style="flex-grow: 1;"
    >
      <div
        class="ant-card-head"
      >
        <div
          class="ant-card-head-wrapper"
        >
          <div
            class="ant-card-head-title"
          >
            Valor total de pagamentos em aberto
          </div>
        </div>
      </div>
      <div
        class="ant-card-body"
      >
        <div
          data-testid="amountPaymentOpen"
        >
          R$ 1.508,15
        </div>
      </div>
    </div>
    <div
      class="ant-card ant-card-bordered css-dev-only-do-not-override-xu9wm8"
      style="flex-grow: 1;"
    >
      <div
        class="ant-card-head"
      >
        <div
          class="ant-card-head-wrapper"
        >
          <div
            class="ant-card-head-title"
          >
            Valor total de pagamentos fechados
          </div>
        </div>
      </div>
      <div
        class="ant-card-body"
      >
        <div
          data-testid="amountPaymentClosed"
        >
          R$ 2.718,15
        </div>
      </div>
    </div>
  </div>
</div>
`;
